apps/www/src/view/searchForm.js

```diff
@@ -31,3 +35,15 @@ function doFilter(template) {
     }))
   }
 }
+
+function searchHasMinLength(template, ptr) {
+  const { value } = ptr.out(hydra.freetextQuery)
+  if (value === '') {
+    return true
```

Comment:
shouldn't this be false?

Reply:
No, because empty is the default, to load all (or first page if/when we add paging)